<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>1091</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutGraph" rowstretch="0,0,0,0">
     <item row="3" column="1">
      <widget class="QLabel" name="labelGraph2">
       <property name="text">
        <string>생산량</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="lblAreaBar">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelGraph1">
       <property name="text">
        <string>불량의 비율</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLabel" name="lblAreaLine">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutControl" stretch="1,1,1,1">
     <item>
      <widget class="QPushButton" name="pushButtonControlStart">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlRestart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>다시시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>일시정지</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonControlStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>정지</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelSingleImageShow">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border-width: 1px;
border-style: solid;
border-color: #000000;</string>
    </property>
    <property name="text">
     <string>원본 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSingleCAMShow">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>170</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border-width: 1px;
border-style: solid;
border-color: #000000;</string>
    </property>
    <property name="text">
     <string>CAM 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSingleStartInference">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>개별 추론</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSingleImageDir">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>140</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px</string>
    </property>
    <property name="text">
     <string>이미지를 선택해주세요.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenSingleImageDir">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>개별 이미지 열기</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetLog">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>991</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget_4" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 3px 5px;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelImageFileDir">
       <property name="styleSheet">
        <string notr="true">font-size: 19px;</string>
       </property>
       <property name="text">
        <string>파일 저장 경로</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textBrowserImageFile">
       <property name="styleSheet">
        <string notr="true">font-size: 17px;</string>
       </property>
       <property name="text">
        <string>이미지 저장 경로를 설정해주세요.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px</string>
    </property>
    <property name="text">
     <string>로그</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserLogContent">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>409</width>
      <height>339</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLog_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>560</y>
      <width>409</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px</string>
    </property>
    <property name="text">
     <string>추론 완료 이미지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLogClear">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelControl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>279</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px</string>
    </property>
    <property name="text">
     <string>모든 이미지 추론 제어</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelListShow">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>840</y>
      <width>279</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px</string>
    </property>
    <property name="text">
     <string>리스트 보기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAllListShow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>870</y>
      <width>277</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;양품, 불품 리스트를 나눠서 볼 수 있습니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>추론 완료 리스트</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSingleImageDir_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>110</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px</string>
    </property>
    <property name="text">
     <string>이미지 위치</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget_5" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>871</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelModelFileDir">
       <property name="styleSheet">
        <string notr="true">font-size: 19px;</string>
       </property>
       <property name="text">
        <string>모델 파일</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textBrowserModelFile">
       <property name="styleSheet">
        <string notr="true">font-size: 17px;</string>
       </property>
       <property name="text">
        <string>모델 파일을 설정해주세요.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIMAGE">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open Log File Directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
